### ISSUE TITLE
Version was removed at some point from IExtensionInfo

### ISSUE THREAD

**Issue by jersiovic** [OrchardCMS/OrchardCore#478]
Was there any reason for removing Version property from IExtensionInfo?
Don't you think version is a useful info for a module.

**Comment by jersiovic** [comment]
I'm using module version for informing consumers of the module of the version of the module. This module is offering a Web Api so it is important it can provide this info.
By the moment I fix it hardcoding the version in the controller. But I think it would be good version would be another time available from IExtensionInfo or from any other way.

**Comment by agriffard** [comment]
We also have to retrieve Author, Version, Web site, ... from manifest files.

**Comment by sfmskywalker** [comment]
I added the following to `IManifestInfo`:

* Tags - `IEnumerable<string>
* Website - `string`
* Author - `string`
* Version - `Version`

As you see, I took the liberty of making the Tags field an enumerable instead of a string (which is how it is in O1), and the Version field of type `System.Version`. I'm thinking having Version as an actual Version type (as opposed to a simple string) might be useful in case we want to compare versions against supported Orchard versions in the future (OrchardVersion not yet added, but when we do, both version fields should be of the same type).

@SebastienRos probably disagrees, but it's easy enough to undo, so I figured I should just do it and see what happens.
